lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -0,0 +1,657 @@
+define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpers/template_helpers',
+        'js/groups/views/cohorts', 'js/groups/collections/cohort', 'js/groups/models/content_group'],
+    function (Backbone, $, AjaxHelpers, TemplateHelpers, CohortsView, CohortCollection, ContentGroupModel) {
+        'use strict';
+
+        describe("Cohorts View", function () {
+            var catLoversInitialCount = 123, dogLoversInitialCount = 456, unknownUserMessage,
+                createMockCohort, createMockCohorts, createMockContentGroups, createCohortsView, cohortsView,
+                requests, respondToRefresh, verifyMessage, verifyNoMessage, verifyDetailedMessage, verifyHeader,
+                expectCohortAddRequest, getAddModal, selectContentGroup, clearContentGroup,
+                MOCK_COHORTED_USER_PARTITION_ID, MOCK_UPLOAD_COHORTS_CSV_URL, MOCK_STUDIO_ADVANCED_SETTINGS_URL,
+                MOCK_STUDIO_GROUP_CONFIGURATIONS_URL;
+
+            MOCK_COHORTED_USER_PARTITION_ID = 0;
+            MOCK_UPLOAD_COHORTS_CSV_URL = 'http://upload-csv-file-url/';
+            MOCK_STUDIO_ADVANCED_SETTINGS_URL = 'http://studio/settings/advanced';
+            MOCK_STUDIO_GROUP_CONFIGURATIONS_URL = 'http://studio/group_configurations';
+
+            createMockCohort = function (name, id, userCount, groupId, userPartitionId) {
+                return {
+                    id: id || 1,
+                    name: name,
+                    user_count: userCount || 0,
```

Comment:
Why are we expecting that `userCount` could be undefined, but subsequent arguments are defined?